easytab.h

```diff
@@ -870,21 +879,41 @@ EasyTabResult EasyTab_Load_Ex(HWND Window,
 
 EasyTabResult EasyTab_HandleEvent(HWND Window, UINT Message, LPARAM LParam, WPARAM WParam)
 {
+    static bool LeftContext = true;
     PACKET Packet = { 0 };
-
     if (Message == WT_PACKET &&
         (HCTX)LParam == EasyTab->Context &&
         EasyTab->WTPacket(EasyTab->Context, (UINT)WParam, &Packet))
     {
         POINT Point = { 0 };
         Point.x = Packet.pkX;
         Point.y = Packet.pkY;
-        ScreenToClient(Window, &Point);
+        if(LeftContext && EasyTab->IsRelative)
+        {
+            Point.x = 0;
+            Point.y = 0;
+            LeftContext = false;
+        }
+        if(!EasyTab->IsRelative)
+        {
+            ScreenToClient(Window, &Point);
+        }
         EasyTab->PosX = Point.x;
         EasyTab->PosY = Point.y;
-
+        EasyTab->Buttons = Packet.pkButtons;
```

Comment:
Ah. You're right. I missed why you set the PACKETMODE to 0 earlier. I don't think we need to overengineer at this point and give ButtonPressed/ButtonReleased information. Whether a given button is pressed or not should be enough.

I'll add the pressed/release feature set if I ever need it later in Papaya.

I'm having some difficulty locating the pkButtons integer interpretation in the docs. Is this like a bitfield with boolean info for 32 different buttons?